### PR TITLE
Add registry sweep documentation to cirq_google

### PR DESCRIPTION
**Pull request by dstrain115**
- Add some simple documentation on how to use registry sweeps to cirq_google's device page (currently public but not linked to from table of contents)

### CODE REVIEW

File: docs/google/devices.md [hunk 1]
```diff
@@ -70,6 +70,35 @@ of the time.  To minimize the duration of the circuit, it is best to align
...
+
+Certain device parameters can be changed for the duration of
+a circuit in order to support hardware parameter sweeps.  For instance,
+frequencies, amplitudes, and various other parameters can be modified
+in order to find optimal values or explore the parameter space.
+
+These parameter names are generally not public, so you will need to
+work with a Google sponsor or resident in order to access the proper
+key names.  These parameters are specified as lists of strings representing
+a path from the device config's folder (or the "sample folder").`
```
Comment (pavoljuhas) [inline docs/google/devices.md:83]:
```suggestion
a path from the device config's folder (or the "sample folder").
```

File: docs/google/devices.md [hunk 2]
```diff
@@ -70,6 +70,35 @@ of the time.  To minimize the duration of the circuit, it is best to align
...
+
+These keys can be swept like any other symbol using the
+`cirq_google.study.DeviceParameter` variable.  For instance, the
+following code will sweep qubit (4,8)'s pi amplitude from 0.0 to 1.0
+in 0.02 increments.
+
+
+```
+descriptor = cirq_google.study.DeviceParameter( ["q4_8", "piAmp"])
+sweep = cirq.Linspace("q4_8.piAmp", 0,1,51, metadata = descriptor)
```
Comment (pavoljuhas) [inline docs/google/devices.md:93]: maybe use standard Python formatting here?
```suggestion
sweep = cirq.Linspace("q4_8.piAmp", 0, 1, 51, metadata=descriptor)
```

**Review by pavoljuhas — CHANGES_REQUESTED** [review]
LGTM, but we can fix a few typos in the document, please see the attached patch.
[typos-in-google-devices.md.patch.txt](https://github.com/quantumlib/Cirq/files/12652299/typos-in-google-devices.md.patch.txt)

**Comment by dstrain115** [comment]
@pavoljuhas Is there some issue with your CLA or github registration?  It's having a problem since I committed your suggestions.

**Comment by pavoljuhas** [comment]
> @pavoljuhas Is there some issue with your CLA or github registration? It's having a problem since I committed your suggestions.

AFAICT, I should be all configured for the CLA check.  The problem seems in commits 1a76baf9183c6a2fda8e467c2fa28dab522af862, ee440ab238ca42018fca2889500cb5ea7c413548 which have `Co-authored-by: ...` note in the commit message.  I have requested CLA override which seems to clear the check here.

**Review by pavoljuhas — APPROVED** [review]

**Pull request merged**